public/app/features/expressions/components/SqlExpressions/SqlExpr.tsx

```diff
@@ -236,168 +201,126 @@ LIMIT
-          {shouldShowViewExplanation ? (
-            <Button
-              fill="outline"
-              icon="gf-movepane-right"
-              onClick={handleOpenExplanation}
-              size="sm"
-              variant="secondary"
-            >
-              <Trans i18nKey="sql-expressions.view-explanation">View explanation</Trans>
-            </Button>
+  const renderButtons = () => (
+    <Stack direction="row" alignItems="center" justifyContent="space-between" wrap>
+      <SqlQueryActions
+        executeQuery={executeQuery}
+        currentQuery={query.expression || ''}
+        queryContext={queryContext}
+        refIds={vars}
+        initialQuery={initialQuery}
+        errorContext={errorContext}
+      />
+      {isSchemasFeatureEnabled && (
+        <Button
+          icon={isSchemaInspectorOpen ? 'table-collapse-all' : 'table-expand-all'}
+          onClick={() => setIsSchemaInspectorOpen(!isSchemaInspectorOpen)}
+          size="sm"
+          variant="secondary"
+          fill="outline"
+        >
+          {isSchemaInspectorOpen ? (
+            <Trans i18nKey="expressions.sql-schema.close-schema-inspector">Close schema inspector</Trans>
           ) : (
-            <GenAISQLExplainButton
-              currentQuery={query.expression || ''}
-              onExplain={handleExplain}
-              queryContext={queryContext}
-              refIds={vars}
-              // schemas={schemas} // Will be added when schema extraction is implemented
-            />
+            <Trans i18nKey="expressions.sql-schema.open-schema-inspector">Open schema inspector</Trans>
           )}
-        </Suspense>
-        <Suspense fallback={null}>
-          <GenAISQLSuggestionsButton
-            currentQuery={query.expression || ''}
-            initialQuery={initialQuery}
-            onGenerate={() => {}} // Noop - history is managed via onHistoryUpdate
-            onHistoryUpdate={handleHistoryUpdate}
-            queryContext={queryContext}
-            refIds={vars}
-            errorContext={errorContext} // Will be added when error tracking is implemented
-            // schemas={schemas} // Will be added when schema extraction is implemented
-          />
-        </Suspense>
-      </Stack>
-      {suggestions.length > 0 && (
-        <Suspense fallback={null}>
-          <SuggestionsDrawerButton handleOpenDrawer={handleOpenDrawer} suggestions={suggestions} />
-        </Suspense>
+        </Button>
       )}
-    </div>
+    </Stack>
   );
 
-  const renderSQLEditor = (width?: number, height?: number) => (
-    <>
-      <div className={styles.sqlContainer}>
-        {renderSQLButtons()}
-        <div
-          className={cx(styles.contentContainer, {
-            [styles.contentContainerWithSchema]: isSchemaInspectorOpen && isSchemasFeatureEnabled,
-          })}
-        >
-          <div ref={containerRef} className={styles.editorContainer}>
+  const renderMainContent = (width?: number, height?: number) => (
```

Comment:
Can remove unused function params! 

```suggestion
  const renderMainContent = () => (
```